In [1]:
import pandas as pd
import matplotlib.pyplot as plt

df = pd.read_csv('nba_logreg2.csv', sep = ";")

# Descripción de los datos
df.describe().T

count       mean        std   min   25%   50%   75%    max
GP           1329.0  60.402558  17.481092  11.0  47.0  63.0  77.0   82.0
MIN          1329.0  17.647931   8.327441   3.1  10.8  16.1  22.9   40.9
PTS          1329.0   6.820166   4.365671   0.7   3.7   5.6   8.8   28.2
FGM          1329.0   2.634763   1.686850   0.3   1.4   2.1   3.4   10.2
FGA          1329.0   5.902483   3.598820   0.8   3.3   4.8   7.5   19.8
FG%          1329.0  44.117532   6.119492  23.8  40.2  43.9  47.8   73.7
3P Made      1329.0   0.249661   0.384608   0.0   0.0   0.1   0.4    2.3
3PA          1329.0   0.785628   1.063856   0.0   0.0   0.3   1.2    6.5
3P%          1329.0  19.308126  16.022916   0.0   0.0  22.4  32.5  100.0
FTM          1329.0   1.302784   0.989341   0.0   0.6   1.0   1.7    7.7
FTA          1329.0   1.827314   1.326655   0.0   0.9   1.5   2.3   10.2
FT%          1329.0  70.407148  10.489838   0.0  64.8  71.3  77.6  100.0
OREB         1329.0   1.007600   0.779178   0.0   0.4   0.8   1.4    5.3
DREB         1329.0   2.021445   1.362464   0.2   1.0   1.7   2.6    9.6
REB          1329.0   3.028442   2.062357   0.3   1.5   2.5   4.0   13.9
AST          1329.0   1.558992   1.474159   0.0   0.6   1.1   2.0   10.6
STL          1329.0   0.620391   0.410533   0.0   0.3   0.5   0.8    2.5
BLK          1329.0   0.366140   0.428534   0.0   0.1   0.2   0.5    3.9
TOV          1329.0   1.196012   0.724470   0.1   0.7   1.0   1.5    4.4
TARGET_5Yrs  1329.0   0.621520   0.485191   0.0   0.0   1.0   1.0    1.0

# el percentil 25 (o primer cuartil) para la variable MIN (minutos) es 10.8, lo que significa que el 25% de los jugadores jugó 10.8 minutos por partido o menos.
# el mínimo de juegos jugados (GP) es 11 y el máximo es 82

In [2]:
df.head()

Name  GP   MIN  PTS  FGM  FGA   FG%  3P Made  3PA   3P%  ...  \
0   Brandon Ingram  36  27.4  7.4  2.6  7.6  34.7      0.5  2.1  25.0  ...   
1  Andrew Harrison  35  26.9  7.2  2.0  6.7  29.6      0.7  2.8  23.5  ...   
2   JaKarr Sampson  74  15.3  5.2  2.0  4.7  42.2      0.4  1.7  24.4  ...   
3      Malik Sealy  58  11.6  5.7  2.3  5.5  42.6      0.1  0.5  22.6  ...   
4      Matt Geiger  48  11.5  4.5  1.6  3.0  52.4      0.0  0.1   0.0  ...   

   FTA   FT%  OREB  DREB  REB  AST  STL  BLK  TOV  TARGET_5Yrs  
0  2.3  69.9   0.7   3.4  4.1  1.9  0.4  0.4  1.3          0.0  
1  3.4  76.5   0.5   2.0  2.4  3.7  1.1  0.5  1.6          0.0  
2  1.3  67.0   0.5   1.7  2.2  1.0  0.5  0.3  1.0          0.0  
3  1.3  68.9   1.0   0.9  1.9  0.8  0.6  0.1  1.0          1.0  
4  1.9  67.4   1.0   1.5  2.5  0.3  0.3  0.4  0.8          1.0  

[5 rows x 21 columns]

In [3]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 1329 entries, 0 to 1328
Data columns (total 21 columns):
 #   Column       Non-Null Count  Dtype  
---  ------       --------------  -----  
 0   Name         1329 non-null   object 
 1   GP           1329 non-null   int64  
 2   MIN          1329 non-null   float64
 3   PTS          1329 non-null   float64
 4   FGM          1329 non-null   float64
 5   FGA          1329 non-null   float64
 6   FG%          1329 non-null   float64
 7   3P Made      1329 non-null   float64
 8   3PA          1329 non-null   float64
 9   3P%          1329 non-null   float64
 10  FTM          1329 non-null   float64
 11  FTA          1329 non-null   float64
 12  FT%          1329 non-null   float64
 13  OREB         1329 non-null   float64
 14  DREB         1329 non-null   float64
 15  REB          1329 non-null   float64
 16  AST          1329 non-null   float64
 17  STL          1329 non-null   float64
 18  BLK          1329 non-null   float64
 19  TOV   